mne/decoding/transformer.py

```diff
@@ -17,7 +18,10 @@
 
 
 class Scaler(TransformerMixin):
-    """Standardize data across channels.
+    u"""Standardize data across channels.
+
+    By default, this makes each time point (within each epoch) have
```

Comment:
> The problem I see with this approach is if there are artifactual epochs, your std dev won't be accurate. You have to use a robust measure ...

I think this is part of the motivation for using `scalings=dict(...)` in `cov` computation. But there being within a couple of orders of magnitude is enough. Here maybe it's more sensitive.

Anyway, back to `std` -- maybe trimmed standard deviation, then?